apps/server/src/modules/tool/context-external-tool/uc/context-external-tool.uc.spec.ts

```diff
@@ -16,7 +16,8 @@ import { ToolPermissionHelper } from '../../common/uc/tool-permission-helper';
 import { SchoolExternalTool } from '../../school-external-tool/domain';
 import { SchoolExternalToolService } from '../../school-external-tool/service';
 import { ContextExternalTool } from '../domain';
-import { ContextExternalToolService, ContextExternalToolValidationService } from '../service';
+import { ContextExternalToolService } from '../service';
+import { ContextExternalToolValidationService } from '../service/context-external-tool-validation.service';
```

Comment:
Use barrel file :)

Reply:
adding the contextExternalToolValidationService back to the index.ts creates dependency cycles. That's why i removed it. We can have a look on it together if you like to